long-term-storage.md

```diff
@@ -1,83 +1,111 @@
 The details of this file are documented thoroughly in [Thanos documentation](https://thanos.io/tip/thanos/storage.md/).
 
+We have guides for using cloud-native storage for the largest cloud providers. Other providers can be similarly configured.
+
 Use the appropriate guide for your cloud provider:
 * [Google Cloud Storage](https://github.com/kubecost/docs/blob/main/long-term-storage-gcp.md)
 * [AWS/S3](https://github.com/kubecost/docs/blob/main/long-term-storage-aws.md)
 * [Azure](https://github.com/kubecost/docs/blob/main/long-term-storage-azure.md)
 
-### Step 2: Create object-store secret
+### <a name="secret"></a>Step 2: Create object-store secret
 
-The final step prior to installation is to create a secret with the yaml file generated in the previous step:
+Create a secret with the yaml file generated in the previous step:
 
 ```shell
 kubectl create secret generic kubecost-thanos -n kubecost --from-file=./object-store.yaml
 ```
 
-### Step 3: Deploying Kubecost with Thanos
+### <a name="cluster_id"></a>Step 3: Unique Cluster ID
+
+Each cluster needs to be labelled with a unique Cluster ID, this is done in two places.
+
+`values-clusterName.yaml`
+```yaml
+kubecostProductConfigs:
+  projectID: yourClusterName
+prometheus:
+  server:
+    global:
+      external_labels:
+        cluster_id: yourClusterName
+```
+
+### <a name="deploy"></a>Step 4: Deploying Kubecost with Thanos
 
-The Thanos subchart includes `thanos-bucket`, `thanos-query`, `thanos-store`,  `thanos-compact`, and service discovery for `thanos-sidecar`. These components are recommended when deploying Thanos on multiple clusters.
+The Thanos subchart includes `thanos-bucket`, `thanos-query`, `thanos-store`,  `thanos-compact`, and service discovery for `thanos-sidecar`. These components are recommended when deploying Thanos on the primary cluster.
 
-These values can be adjusted under the `thanos` block in `values-thanos.yaml` - Available options can be observed here: [thanos/values.yaml](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/charts/thanos/values.yaml)
+These values can be adjusted under the `thanos` block in `values-thanos.yaml` - Available options are here: [thanos/values.yaml](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/charts/thanos/values.yaml)
 
-It's *important* to note that when running `helm install`, you must provide the base `values.yaml` followed by the override [values-thanos.yaml](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values-thanos.yaml). For example:
 
 ```shell
 helm upgrade kubecost kubecost/cost-analyzer \
     --install \
     --namespace kubecost \
-    -f values.yaml \
-    -f values-thanos.yaml
+    -f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/master/cost-analyzer/charts/thanos/values.yaml \
```

Comment:
Shouldn't this be https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/master/cost-analyzer/values-thanos.yaml ?

Reply:
great catch